src/program/UplcProgram.js

```diff
@@ -154,7 +157,12 @@ export function evalProgram(builtins, costModel, expr, args) {
         }
     }
 
-    const machine = new CekMachine(expr, builtins, costModel)
+    const machine = new CekMachine(
+        expr,
+        builtins,
+        costModel,
+        logOptions || new BasicUplcLogger()
```

Comment:
Do you think it would be a good idea to use `??` instead of `||`?

I've been using the nullish coalescing operator (`??`) throughout for default values, and I think `??` has sufficient support to be considered standard.

Reply:
I find it makes sense when the left thing can be false or 0 ("provided, but falsey") and we only want a fallback when it's not provided - it's great for that because the syntax itself makes you question (pun intended) what's going on.

When the left thing can't be like that, the `||` syntax reads more clearly to me.  

I'm okay changing it to `??` if the consistency is high value for you.

Reply:
Hadn't thought about it like!

So to conclude: only use `??` for Option<falsy-type> if the falsy value itself doesn't require a fallback

I'll keep them this in mind from now on